pkg/controller/account/iam.go

```diff
@@ -45,7 +45,9 @@ type StatementEntry struct {
 }
 
 var (
-	DefaultDelay = 3 * time.Second
+	DefaultDelay          = 3 * time.Second
+	DefaultSleepDelay     = 500 * time.Millisecond
+	DefaultScale      int = 1
```

Comment:
`DefaultScale` is used for what (I can see where it is used in code, but what semantic meaning)?

Reply:
Given the way it's used in the code:
```
time.Sleep(time.Duration(time.Duration(attempt*5*DefaultScale) * time.Second))
```
`attempt` is an int which increases per loop attempt, so I couldn't just set the overall sleep time for this - so I decided to add a multiplier scale factor which I would set to 0 for tests, while remaining at 1 for the normal use it shouldn't affect the previous implementation

Reply:
- I'm happy to rename if you have a different idea
- I can also leave a comment to further describe it

Reply:
Renamed `DefaultScale` to `TestSleepModifier`